client/lib/plans/features-list.js

```diff
@@ -104,7 +104,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'All Premium features' ),
 		getDescription: () =>
 			i18n.translate(
-				'Including unlimited premium themes, advanced design and monetization options, simple payment buttons, and a custom domain name for one year.'
+				'Including unlimited premium themes, advanced design and monetization options, Pay with PayPal buttons, and a custom domain name for one year.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.